src/main/scala/nl/biopet/utils/tool/ToolCommand.scala

```diff
@@ -145,19 +145,34 @@ trait ToolCommand[Args] extends Logging {
   def example(args: String*): String = {
     cmdArrayToArgs(args.toArray)
 
-    exampleToMarkdown(args: _*)
+    exampleToMarkdown(false, args: _*)
   }
 
   /** Convert and *not* tests args */
   def unsafeExample(args: String*): String = {
-    exampleToMarkdown(args: _*)
+    exampleToMarkdown(false, args: _*)
+  }
+
+  /** Convert and tests args */
+  def sparkExample(args: String*): String = {
+    cmdArrayToArgs(args.toArray)
+
+    exampleToMarkdown(true, args: _*)
+  }
+
+  /** Convert and *not* tests args */
+  def sparkUnsafeExample(args: String*): String = {
+    exampleToMarkdown(true, args: _*)
   }
 
   /** Common function to convert to string */
-  private def exampleToMarkdown(args: String*): String = {
+  private def exampleToMarkdown(spark: Boolean, args: String*): String = {
```

Comment:
I do not like that this spark value is required. It's better to use 
```Scala
(args: String*, spark: Boolean = false)
```
instead.

Reply:
Not possible, * arguments must be the last